cluster-state-service/internal/features/e2etasksteps/common_steps.go

```diff
@@ -29,7 +29,7 @@ type Exception struct {
 
 var (
 	// Lists to memorize results required for the subsequent steps
-	ecsTaskList   = []ecs.Task{}
+	EcsTaskList   = []ecs.Task{}
```

Comment:
Instead of doing this, let's just add the var ecsTaskList to e2einstancesteps/common_steps.go and use that for instance tests. The idea is to keep the instance and task feature step implementations separate.